README.md

```diff
@@ -137,17 +137,17 @@ Once Prisma Client was generated, you can import in your code and send queries t
 You can import and instantiate Prisma Client as follows:
 
 ```ts
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient } from "@prisma/client";
```

Comment:
unrelated and unwanted change?

Reply:
Ah - unwanted. My formatter did that. Will revert.

Reply:
There are many of those - just commented on this one.